setup/src/Magento/Setup/Test/Unit/Model/ConfigOptionsList/CacheTest.php

```diff
@@ -36,6 +39,9 @@ protected function setUp()
         $this->configOptionsList = new CacheConfigOptionsList($this->validatorMock);
     }
 
+    /**
+     * testGetOptions
```

Comment:
@nmalevanec why need to add such useless annotations? Tests were ignored before, what caused such a bug?

Reply:
@orlangur, I've moved this comment from 2.3-develop. But, ok I'll remove it.

Reply:
@nmalevanec no need to remove, they must be not needed in `2.3-develop` as well.